### DISCUSSION TITLE
navigation menu not display verticla scroll bar when expand the menu

### DISCUSSION THREAD

**Discussion by abualmazen** [serenity-is/Serenity#6829] [General]
navigation menu not display vertical scroll bar when expand the menu sow when i expand one of the navigation menu i can't reach the the other navigation item at th bottom
how i can fixed pleas?

**Comment by wezmag** [comment]
Check out this discuss. https://github.com/orgs/serenity-is/discussions/6753

**Comment by abualmazen** [comment]
I found the solution in the link you sent
thank you
